src/Shared/FileSystem/MSBuildOnWindowsFileSystem.cs

```diff
@@ -59,6 +60,11 @@ public FileAttributes GetAttributes(string path)
             return ManagedFileSystem.Singleton().GetAttributes(path);
         }
 
+        public DateTime GetLastWriteTimeUtc(string path)
+        {
+            return ManagedFileSystem.Singleton().GetLastWriteTimeUtc(path);
```

Comment:
I'll replace this with WindowsFileSystem in a later PR when I actually hook this method up in the evaluator